scripts/workers/remote_submission_worker.py

```diff
@@ -576,6 +580,15 @@ def run_submission(
     else:
         status = "failed"
         submission_data["submission_status"] = status
+
+    if challenge_phase.get("challenge").get("is_docker_based"):
+        sender_email = settings.CLOUDCV_TEAM_EMAIL
+        email = submission.get("created_by").get("email")
```

Comment:
@takitsuse is this the user email? If yes can we rename the variable it is confusing. I though this was email body